x-pack/platform/test/serverless/functional/test_suites/discover/group1/_discover.ts

```diff
@@ -109,8 +109,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         const time = await PageObjects.timePicker.getTimeConfig();
         // TODO: The Serverless sidebar causes `PageObjects.discover.clickHistogramBar()`
         // to click a different range in the histogram, resulting in a different duration
-        expect(time.start).to.be('Sep 19, 2015 @ 06:31:44.000');
-        expect(time.end).to.be('Sep 23, 2015 @ 18:31:44.000');
```

Comment:
I think the new sidebar width is smaller, which pushes Discover less, so the click on the histogram happens in a different place.